src/Desktop/UIAutomation/A11yAutomation.cs

```diff
@@ -136,17 +150,17 @@ public static IEnumerable<DesktopElement> ElementsFromProcessId(int pid, Registr
         /// <summary>
         /// Get DesktopElement from UIAElement interface.
         /// </summary>
-        /// <param name="uia"></param>
-        /// <returns></returns>
-        private static DesktopElement ElementFromUIAElement(IUIAutomationElement uia, Registrar registrar)
+        private DesktopElement ElementFromUIAElement(IUIAutomationElement uia, DesktopDataContext dataContext)
         {
+            EnsureContextConsistency(dataContext);
```

Comment:
nit: it looks like this is only called from ElementsFromProcessId, which already called EnsureContextConsitency, so this seems to be redundant